### PR TITLE
Use Three component in localisation

### PR DESCRIPTION
**Pull request by BrendanAnnable**

### CODE REVIEW

File: src/client/components/three/computed.ts [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+) => (expr: () => O): IComputedValue<T> => {
+  let instance: T
+  return computedFn(() => {
+    const opts = expr()
+    if (instance == null) {
+      instance = create(opts)
+    }
+    update(instance, opts)
+    return instance
+  }, { equals: () => false })
```
Comment (BrendanAnnable) [inline src/client/components/three/computed.ts]:
```ts
{ equals: () => false }
```

This is the magic I was missing. If I thought about this earlier I wouldn't have bothered with the reconciler 😄 

This allows consumers to recompute even if we return the same instance.

Should this ever return true? Probably not.

If this `equals` method is ever called, it means an observable dependency has changed, which means a 99% chance that we no longer equal. So there is little to zero reason to waste CPU cycles confirming that we don't. It's a win-win!

Comment (TrentHouliston) [reply src/client/components/three/computed.ts]: The only reason I would still thing equals true might be useful is if you have a dependency in your dependency that changed, and you yourself didn't change. In which case... does always returning equals true work too?

File: src/client/components/three/computed.ts [hunk 2]
```diff
@@ -45,7 +45,6 @@ const computedInstance = <T, O>(
 const disposedComputed = <T extends { dispose(): void; }>(fn: () => T): IComputedValue<T> => {
   let latest: T | undefined
   const expr = computed(() => {
-    latest && latest.dispose()
```
Comment (TrentHouliston) [inline src/client/components/three/computed.ts]: Does this even do anything anymore without dispose?
Now it's basically the same as `computed(fn)` right?

Comment (BrendanAnnable) [reply src/client/components/three/computed.ts]: The `dispose` on `onBecomeUnobserved` is still used below.

Comment (BrendanAnnable) [reply src/client/components/three/computed.ts]: improved with c5af825e

Comment (TrentHouliston) [reply src/client/components/three/computed.ts]: At this point it's almost worth using atoms for this

Comment (BrendanAnnable) [reply src/client/components/three/computed.ts]: How would atoms help?

Comment (TrentHouliston) [reply src/client/components/three/computed.ts]: Atoms are designed to manage a resource and notify when things change, aren't they? Which is what we are doing here?

Comment (TrentHouliston) [reply src/client/components/three/computed.ts]: Fair enough

**Pull request merged**